humblegen/src/ast.rs

```diff
@@ -20,6 +25,17 @@ pub enum SpecItem {
     StructDef(StructDef),
     /// `enum` definition.
     EnumDef(EnumDef),
+    /// `service` definition
+    ServiceDef(ServiceDef),
+}
+
+impl SpecItem {
+    pub fn service_def(&self) -> Option<&ServiceDef> {
```

Comment:
Docs please, even if they are simple. Currently everything else in this file is documented.